src/SprykerEco/Shared/Braintree/Transfer/braintree.transfer.xml

```diff
@@ -91,4 +92,18 @@
         <property name="quote" type="Quote"/>
     </transfer>
 
+    <transfer name="PaypalExpressSuccessResponse">
+        <property name="nonce" type="string"/>
+        <property name="email" type="string"/>
+        <property name="firstName" type="string"/>
+        <property name="lastName" type="string"/>
+        <property name="payerId" type="string"/>
+        <property name="recipientName" type="int"/>
+        <property name="line1" type="int"/>
+        <property name="city" type="int"/>
+        <property name="state" type="int"/>
+        <property name="postalCode" type="int"/>
```

Comment:
are you sure that it's int? As I know some postal code can be alphanumeric. Or it's api restriction?